lib/appsignal.rb

```diff
@@ -301,6 +294,15 @@ def start_stdout_logger
         "appsignal: #{msg}\n"
```

Comment:
I think it's quite normal an application would log its own logs with a timestamp. That Heroku adds a timestamp to all logs is something that's part of their logging system and only because it's wrapped in a foreman-like.

I'm 👍  for adding a timestamp to it, not making an exception for Heroku to stay consistent.